test/ACLManager.test.js

```diff
@@ -0,0 +1,164 @@
+const assert = require('assert')
+const { ACLManager, ACLService } = require('./../index')
+const acl1 = new ACLService()
+const acl2 = new ACLService()
+let acm
+
+// Lowest ACL level
+const acl0Config = {
+  "admin": [ // We will never get here
+    "*"
+  ],
+  "operator": [  // Falls back here from acl2 then acl1
+    "calls.accept"
+  ]
+}
+
+// Lower precedence ACL
+acl1.import({
+  "admin": [ // We will never get here
+    "users.*",
+    "system.*"
+  ],
+  "supervisor": [ // Falls back here from acl2
+    "users.*",
+    "!users.delete",
+    "system.*",
+    "!system.shutdown",
+    "@ignored"
+  ]
+})
+
+// Higher precedence ACL
+acl2.import({
+  "admin": [
+    "users.*",
+  ]
+})
+
+describe('ACLManager.constructor', function () {
+  it('Can construct a new ACLManager', function () {
+    acm = new ACLManager()
```

Comment:
Tipp: teszteknél én úgy szoktam, hogy minden teszt esetnél külön hozok létre példányokat.
Így tisztább, és nem kavarodik össze az állapota a dolgoknak tesztek között, nem lesznek false pozitív/negatív tesztek.

Reply:
Egyetértek. Viszont itt egyszer van setup egyszer clear a végén, és közte csak lekérések vannak amik nem módosítanak. Plusz a másik is így volt... Átírjam ezt most, vagy jó lesz így is?

Reply:
Jó lesz így persze